common/zoom/index.ts

```diff
@@ -15,26 +16,40 @@ type JWTPayload = {
     tokenExp: number;
 };
 
+const sdkKey = process.env.ZOOM_MEETING_SDK_CLIENT_ID;
+const secret = process.env.ZOOM_MEETING_SDK_CLIENT_SECRET;
+
+if (isZoomFeatureActive()) {
+    assert(sdkKey, 'Missing ZOOM_MEETING_SDK_CLIENT_ID in ENV');
```

Comment:
Added some assertions to ensure proper configuration

Reply:
Nice one 👍🏻